### DISCUSSION TITLE
Full RTMA-RU archive ?

### DISCUSSION THREAD

**Discussion by jm5791** [blaylockbk/Herbie#234] [Q&A]
Hi,

I recently discovered the "Rapid Update" (i.e. 15 min) version of RTMA, that seems to be operational starting in 2017, according to [this page](https://www.faa.gov/nextgen/programs/weather/awrp). 
However, I was not able to find a full RTMA-RU anywhere so far:
- the [AWS bucket](https://noaa-rtma-pds.s3.amazonaws.com/index.html) seems to only store RTMA-RU from 2021 (note: there are also directories for the period 2019-2021, but, as far as I understand, they only contain satellite winds bufr files ... which I don't really understand, actually)
- the [NCEP server ](https://nomads.ncep.noaa.gov/pub/data/nccf/com/rtma/prod/) only stores the last 10 days.

Anybody aware of a full RTMA-RU archive somewhere else ?

Thanks !

**Comment by omriporat1** [comment]
hey @jm5791,
I'm currently working on a streamflow prediction model and was hoping to include RTMA-RU as an input. I was wondering if you ever found a source for the RTMA-RU data from 2017 onwards? It would help me a lot with my model training.
Thanks!
Omri
